src/main/java/microsoft/exchange/webservices/data/core/request/HangingServiceRequestBase.java

```diff
@@ -80,7 +80,7 @@
    * Test switch to log all bytes that come across the wire.
    * Helpful when parsing fails before certain bytes hit the trace logs.
    */
-  public static boolean LogAllWireBytes = false;
+  public static boolean logAllWireBytes = false;
```

Comment:
maybe this one should be `final` and written `LOG_ALL_WIRE_BYTES` like introduced in [docs.oracle.com](http://docs.oracle.com/javase/specs/jls/se6/html/names.html#6.8.6)